lms/djangoapps/teams/static/teams/js/spec/teams_factory_spec.js

```diff
@@ -1,11 +1,13 @@
-define(["jquery", "teams/js/teams_tab_factory"],
-    function($, TeamsTabFactory) {
+define(["jquery", "backbone", "teams/js/teams_tab_factory"],
+    function($, Backbone, TeamsTabFactory) {
         'use strict';
        
         describe("teams django app", function() {
             var teamsTab;
 
             beforeEach(function() {
+                // Prevent Backbone history from being started more than once
+                spyOn(Backbone.history, 'start');
```

Comment:
I had hoped for that, but alas, it's needed to select the right tab when the user navigates to a specific one in the URL.